doc/api/test.md

```diff
@@ -1521,6 +1521,26 @@ Emitted when all subtests have completed for a given test.
 
 Emitted when a test starts.
 
+### Event: `'test:stderr'`
+
+* `data` {Object}
+  * `file` {string|undefined} The path of the test file,
+    undefined if test is not ran through a file.
```

Comment:
```suggestion
  * `file` {string|undefined} The path of the test file.
```

Reply:
The explanation is the same with other `file` in other events. I prefer to keep them consistent. Or we can rename others.

Reply:
but it isn't consistent. this event only fires when running through a file

Reply:
> this event only fires when running through a file

Wouldn't that mean the type of `file` should be `string`, not `string|undefined`?

Reply:
yes, that is my point

Reply:
Oops, I misunderstood, it makes sense. Done, thanks for explaining!